testdata/baselines/reference/submoduleAccepted/conformance/exportNestedNamespaces2.errors.txt.diff

```diff
@@ -42,4 +44,15 @@
 +!!! error TS2304: Cannot find name 'exports'.
          return v
      }
+     
+-==== use.js (0 errors) ====
++==== use.js (2 errors) ====
+     import * as debug from './mod'
+     
+     debug.formatters.j
++                     ~
```

Comment:
this is an improvement in imports, it's just that multi-file expandos are not supported in Corsa. I'll send a later PR adding to the list of accepted diffs.